packages/gatsby/cache-dir/page-renderer.js

```diff
@@ -133,7 +133,7 @@ class PageRenderer extends React.Component {
 
     const [replacementComponent] = apiRunner(`replaceComponentRenderer`, {
       props,
-      loader: publicLoader,
+      component: this.state.pageResources.component,
```

Comment:
I was just looking how this was done in v1 - https://github.com/gatsbyjs/gatsby/blob/9da74543c1d092fef1b48db79899e26cf4b82726/packages/gatsby/cache-dir/component-renderer.js#L123-L126

Maybe reinstate that here?

Reply:
Hmm yeah, I'll update to that. I wonder why it was changed?